### DISCUSSION TITLE
TV Shows: Support for multiple <studio> tags

### DISCUSSION THREAD

**Discussion by rainerhillebrand** [Komet/MediaElch#1703] [Ideas]
Under the assumption that my understanding of the definitions on the page https://kodi.wiki/view/NFO_files/TV_shows is correct I would assume that multiple <studio> tags are supported in nfo files for TV Shows.

MediaElch seems to support only a single <studio> tag so that multiple studios can only be given in a single string.

When filtering TV Shows on studios in Kodi (at least in Nexus 20.2) then you cannot select one of these multiple studios that are given in a string of multiple studios.
Example content of a <studio> tag: "ZDF, ORF, SRF1"
I want to filter all TV Shows from the studio "SRF1". However, if there is no other TV Show with the <studio> tag content "SRF1" then I cannot select the filter for "SRF1".

Wouldn't it make sense that MediaElch supports multiple <studio> tags for TV Shows nfo files as well?

**Comment by bugwelle** [comment]
Yes, indeed. :)

I created a feature request from this discussion: https://github.com/Komet/MediaElch/issues/1705

**Reply by bugwelle** [reply]
I created a feature request from this discussion: https://github.com/Komet/MediaElch/issues/1705
